webviz_subsurface/plugins/_rft_plotter/_views/_sim_vs_obs_view/_settings/_ensembles.py

```diff
@@ -0,0 +1,26 @@
+from typing import List
+
+import webviz_core_components as wcc
+from dash.development.base_component import Component
+from webviz_config.utils import StrEnum
+from webviz_config.webviz_plugin_subclasses import SettingsGroupABC
+
+
+class Ensembles(SettingsGroupABC):
+    class Ids(StrEnum):
+        ENSEMBLES = "ensembles"
+
+    def __init__(self, ensembles: List[str]) -> None:
+        super().__init__("Ensembles")
+        self._ensembles = ensembles
+
+    def layout(self) -> List[Component]:
+        return [
+            wcc.Dropdown(
+                id=self.register_component_unique_id(self.Ids.ENSEMBLES),
+                options=[{"label": ens, "value": ens} for ens in self._ensembles],
+                value=[self._ensembles[0] if len(self._ensembles) > 0 else None],
```

Comment:
```suggestion
                value=[self._ensembles[0] if self._ensembles else None],
```
...if I'm not mistaken this should be possible and preferred (I'm actually a bit surprised pylint didn't complain - so maybe I'm wrong)

Reply:
Good point!

Reply:
I have fixed that many other places in the plugin as well.